piker/clearing/_client.py

```diff
@@ -160,10 +161,18 @@ async def send_order_cmds():
     book._ready_to_receive.set()
 
     async for cmd in orders_stream:
-
-        # send msg over IPC / wire
-        log.info(f'Send order cmd:\n{pformat(cmd)}')
-        yield cmd
+        print(cmd)
```

Comment:
This is the major hack of note 😂 

We need proper broadcast channels as mentioned in https://github.com/goodboy/tractor/issues/204 to avoid this re-insertion blasphemy.